modules/op-understanding-credential-selection.adoc

```diff
@@ -0,0 +1,42 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-using-git-secret.adoc
+
+[id="op-understanding-credential-selection_{context}"]
+= Understanding credential selection
+
+A pipeline run or task run may require multiple authentications for accessing different Git repositories. You must annotate each secret to specify the domains for which {pipelines-shortname} can use the credential contained in a  secret.
+
+A credential annotation key for Git secrets must begin with `tekton.dev/git-`, and its value is the URL of the host for which you want {pipelines-shortname} to use that credential.
+
+For example, {pipelines-shortname} uses a `basic-auth` secret (username and password) to access repositories at `github.com` and `gitlab.com`.
```

Comment:
```suggestion
In the following example, {pipelines-shortname} uses a `basic-auth` secret, which relies on a username and password, to access repositories at `github.com` and `gitlab.com`.
```
IBM SG, in the "Parentheses" sections, states "Use parentheses to identify items such as abbreviations, symbols, and measurements, but avoid using parentheses in general text."
Here, I've replaced the parentheses with a parenthetical phrase between commas. You could also remove that phrase altogether.